Makefile

```diff
@@ -1,7 +1,11 @@
-CPPFLAGS += -I /usr/lib/kframework/include -I dummy-version -I plugin -I plugin-c -I libff/build/install/include -I deps/cpp-httplib
+# set default install prefix if not set
+PREFIX ?= $(CURDIR)/build
+LOCAL_INCLUDE := $(PREFIX)/include
+
+CPPFLAGS += -I /usr/lib/kframework/include -I $(LOCAL_INCLUDE) -I dummy-version -I plugin -I plugin-c -I deps/cpp-httplib
```

Comment:
This works for global system installs of K on _Linux_ systems, but not for an install of K somewhere else. This is important for making it easier for developers to use a custom K (usually the submodule they pin) when testing changes to K.

Maybe a better way is to say:

```
K_RELEASE   ?= $(dir $(shell which kompile))../lib/kframework
K_BIN       := $(K_RELEASE)/bin
K_LIB       := $(K_RELEASE)/lib

CPPFLAGS += -I $(K_RELEASE)/include
```

@dwightguth thoughts?